utils/vibe/internal/conv.d

```diff
@@ -0,0 +1,22 @@
+module vibe.internal.conv;
+
+import std.traits : OriginalType;
+
+string enumToString(E)(E value)
+{
+	import std.conv : to;
+
+	switch (value) {
+		default: return "cast("~E.stringof~")"~(cast(OriginalType!E)value).to!string;
+		static foreach (m; __traits(allMembers, E)) {
+			static if (!isDeprecated!(E, m)) {
+				case __traits(getMember, E, m): return m;
+			}
+		}
+	}
+}
+
+private enum isDeprecated(alias parent, string symbol) =
+	!__traits(compiles, {
+		static assert(!__traits(isDeprecated, __traits(getMember, parent, symbol)));
+	});
```

Comment:
Why isn't this just: ```d
private enum isDeprecated(alias parent, string symbol) = __traits(isDeprecated, __traits(getMember, parent, symbol));
```
In my PR, just using that mechanism didn't trigger the deprecation. Honestly, I don't know why you need this wrapper here.

Reply:
Also, this is fine for enums without repeat values, but will not work for ones with repeat values. It might be OK if this is just for internal use.

Reply:
Okay, testing on run.dang.io, this has apparently been fixed in 2.089.0. We support 2.090.1 onwards, so I'll remove this workaround.

Reply:
> It might be OK if this is just for internal use.

That's the case here.